doc/posts/zitification/ssh/zitifying-ssh.md

```diff
@@ -0,0 +1,133 @@
+is no longer accessible directly through a network it is no longer susceptible to the types of attacks mentioned
+previously!
+
+<hr>
+
+## Zssh in Action
+
+Once the prerequisites are satisfied, we can see `zssh` in action. Simply download the binary for your platform:
+
+* [linux](https://github.com/openziti-incubator/zssh/releases/download/latest-tag/zssh-linux-amd64)
+* [windows](https://github.com/openziti-incubator/zssh/releases/download/latest-tag/zssh-windows-amd64.exe)
+* [MacOs](https://github.com/openziti-incubator/zssh/releases/download/latest-tag/zssh-macos-amd64)
+
+Once you have the executable download, make sure it is named `zssh` and for simplicity's sake we'll assume it's on the
+path. The usage for `zssh` is very similar to the usage of `ssh` so anyone familiar with `zssh` should be able to pick
+up `zssh` easily, Should you forget the usage, executing the binary with no arguments returns the expected usage as
+well. The general format when uzing `zssh` will be: `zssh <remoteUsername>@<targetIdentity>`
+
+Below you can see me `zssh` from my local machine to the AWS machine secured by `ziti-tunnel`:
+
+    ./zssh ubuntu@ziti-tunnel-aws
+    INFO[0000] connection to edge router using token 95c45123-9415-49d6-930a-275ada9ae06f
+    connected.
+    ubuntu@ip-172-31-27-154:~$
+
+It really was that simple! Now let's break down the current flags for `zssh` and exactly how this worked.
+
+<hr>
+
+## Zssh Flags
+
+We know that `zssh` requires access to
+a [Ziti Network](https://openziti.github.io/ziti/overview.html#overview-of-a-ziti-network) but it is not clear from the
+example above is where `zzsh` found the credentials required to access that network. `zssh` supports three basic flags:
+
+    -i, --SshKeyPath string   Path to ssh key. default: $HOME/.ssh/id_rsa
+    -c, --ZConfig string      Path to ziti config file. default: $HOME/.ziti/zssh.json
+    -d, --debug               pass to enable additional debug information
+
+What you see above is exactly the output `zssh` provides should you pass the `-h/--help` flag or execute `zssh`
+without any parameters. The `-i/--SshKeyPath` flag is congruent to the `-i` flag for `ssh`. You would use it to supply
+your key to the `ssh` client. Under the hood of `zssh` is a full-fledged `ssh` client which works just like `ssh` does.
+If your `~/.ssh/id_rsa` file is in the `authorized_keys` of the remote machine - then you won't need to specify the
+`-i/` flag (as I didn't in my example). It is required to use a key when using `zssh`.
+
+The `-c/--ZConfig` flag controls access to the network. It is also required when using `zssh`. By default `zssh`
+will look at your home directory in a folder named `.ziti` for a file named `zssh.json`. In bash this is would be the
+equivalent of `$HOME`. In powershell this is the equivalent the environment variable named `USERPROFILE`. If that file
```

Comment:
For Powershell, it is `$env:USERPROFILE`

Reply:
Also `PowerShell` not `powershell`

Reply:
i meant to change PowerShell to Windows - which I've done. thx